IntegrationTests/Services/AWSS3IntegrationTests/S3FlexibleChecksumsTests.swift

```diff
@@ -81,6 +82,46 @@ final class S3FlexibleChecksumsTests: S3XCTestCase {
         XCTAssertEqual(data, originalData)
     }
 
+    // Streaming unsigned payload without checksum (chunked encoding)
+    class DisablePayloadSigning<InputType, OutputType>: HttpInterceptor {
```

Comment:
Should this interceptor just be something we provide to customers?

Reply:
We could, as part of some new enum that contains common interceptors.